openscap_daemon/system.py

```diff
@@ -640,6 +640,17 @@ def generate_report_for_task_result(self, task_id, result_id):
             self.config
         )
 
+    def generate_fix_for_task_result(self, task_id, result_id, fix_type):
+        task = None
+        with self.tasks_lock:
+            task = self.tasks[task_id]
+
+        return task.generate_fix_for_result(
```

Comment:
@mpreisler I think no, but I'm not sure. The method shouldn't change the task, because it reads an ARF from 1 task result and generates a bash script, it doesn't write into the task. Also, we do it same way in `get_xml_of_task_result`, `generate_report_for_task_result`, `get_stdout_of_task_result` etc. What reason for locking this are you thinking about?